addOns/network/src/main/java/org/zaproxy/addon/network/ExtensionNetwork.java

```diff
@@ -541,7 +589,36 @@ public void hook(ExtensionHook extensionHook) {
             optionsDialog.addParamPanel(networkNode, clientCertificatesOptionsPanel, true);
             hookView.addOptionPanel(
                     new LegacyOptionsPanel("clientcerts", clientCertificatesOptionsPanel));
+
+            rateLimitIcon =
+                    new ImageIcon(ExtensionNetwork.class.getResource("resources/ratelimit.png"));
+            rateLimitOptionsPanel = new RateLimitOptionsPanel();
+            optionsDialog.addParamPanel(networkNode, rateLimitOptionsPanel, true);
```

Comment:
The panel is being added with this statement, just don't add the legacy.

Reply:
I tried. When clicking on the rate limit item under network the panel doesn’t show. It does not change from whatever was the previous panel.

Reply:
Are there any other local changes? Works fine here when the legacy statement is removed.

Reply:
Worked too many hours yesterday. It's working.